### PR TITLE
Fixed undefined white decoration radar colors

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes https://github.com/OpenRA/OpenRA/issues/8855.

### CODE REVIEW

**Comment by pchote** [comment]
Looks much better.  Thanks :+1:

File: mods/ts/tilesets/snow.yaml [hunk 1]
```diff
@@ -55,6 +55,14 @@ Terrain:
 		Type: Veins
 		Color: 226, 103, 0
 		TargetTypes: Ground
+	TerrainType@Tree:
+		Type: Tree
+		TargetTypes: Ground
+		Color: 255,28,32,36
```
Comment (pchote) [inline mods/ts/tilesets/snow.yaml]: These dark colours look quite out of place on the bright snow maps.

Comment (Mailaender) [inline mods/ts/tilesets/snow.yaml]: Propose an alternative. (It is the value from https://github.com/OpenRA/OpenRA/blob/bleed/mods/cnc/tilesets/snow.yaml#L49)

**Comment by pchote** [comment]
This is still an improvement over what we have now, so we can adjust the snow terrain colors later. :+1:

**Comment by obrakmann** [comment]
:+1:

**Pull request merged**